MmSupervisorPkg/Core/Handler/Mmi.c

```diff
@@ -341,18 +341,21 @@ MmiUserHandlerRegister (
 }
 
 /**
-  Unregister a handler in MM.
+  Unregister a handler in MM with specified ownership.
 
-  @param  DispatchHandle  The handle that was specified when the handler was registered.
+  @param  DispatchHandle        The handle that was specified when the handler was registered.
+  @param  IsSupervisorHandler   The flag to indicate this is intending to unregister user or
+                                supervisor handler.
 
   @retval EFI_SUCCESS           Handler function was successfully unregistered.
   @retval EFI_INVALID_PARAMETER DispatchHandle does not refer to a valid handle.

```

Comment:
You could update the `EFI_INVALID_PARAMETER` parameter description.

Reply:
Thanks, updated!